scss/custom/_reset.scss

```diff
@@ -318,6 +318,10 @@ input[type="month"] {
   // and https://github.com/twbs/bootstrap/issues/11266
   -webkit-appearance: listbox;
 }
+input[type="email"] {
+  -moz-appearance: none;
```

Comment:
This removes all input styling which isn't really want we want:

<img width="632" alt="Screen Shot 2021-05-24 at 4 15 58 PM" src="https://user-images.githubusercontent.com/9326234/119408486-68e1f380-bcab-11eb-9ed7-76e5b9f74f95.png">

What's the issue you're trying to fix? Firefox looks good on my end.

Reply:
The email input in Firefox has a spacing issue, it should be as wide as the message textarea.
Firefox:
![Screen Shot 2021-05-24 at 4 22 57 PM](https://user-images.githubusercontent.com/68612748/119409473-fbcf5d80-bcac-11eb-8b9f-f3437e027760.png)

Chrome:
![Screen Shot 2021-05-24 at 4 27 01 PM](https://user-images.githubusercontent.com/68612748/119409502-05f15c00-bcad-11eb-9e8a-009e85a898c4.png)

Reply:
I'm seeing it now after a Firefox update.

Want to try setting `border: 0` instead?